pkg/kapp/diff/change_set_with_versioned_rs.go

```diff
@@ -76,6 +84,72 @@ func (d ChangeSetWithVersionedRs) Calculate() ([]Change, error) {
 	return allChanges, nil
 }
 
+func (d ChangeSetWithVersionedRs) checkAndUpdateResWithMaxDurationAnn(existingRsGrouped map[string][]ctlres.Resource,
```

Comment:
Let's move this to another file scoped to a struct, something like `ResourcesWithRenewedResources`